packages/rfw/lib/src/dart/binary.dart

```diff
@@ -521,7 +521,7 @@ class _BlobEncoder {
   }
 
   void _writeString(String value) {
-    final Uint8List buffer = utf8.encode(value) as Uint8List;
+    final Uint8List buffer = const Utf8Encoder().convert(value);
```

Comment:
Is the old code going to stop working/start erroring at some point for end users?

If this needs to be published to pub so users stop using the old version, this package needs an update to pubspec.yaml and CHANGELOG (as described by the repo_checks check).

If not, we can add a label to skip those (but it might be nice to describe this as a `## NEXT` entry in the changelog)

Reply:
> If not, we can add a label to skip those (but it might be nice to describe this as a ## NEXT entry in the changelog)

Not quite sure I follow here. Are you saying you prefer suppressing the analyzer warning instead of doing this CL?

Seems to me that it's better to have code that doesn't need analyzer suppressions if possible (which it is here).

Not quite sure I understand why it would require an update to CHANGELOG.md - as this is just changing some of the inner workings of the package to avoid an explicit downcast (doesn't seem relevant to users).

Reply:
Please see https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates for this repository's policy on versioning and changelogs.